mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -158,6 +159,13 @@ sealed trait Matrix extends Serializable {
    */
   @Since("1.5.0")
   def numActives: Int
+
+  /**
+   * Convert this matrix to the new mllib-local representation.
+   * This does NOT copy the data; it copies references.
+   */
+  private[spark]
+  def toNew: newlinalg.Matrix
```

Comment:
I think `asML` will make more sense since they actually share the same data structure, and `to` is a little confusing?